processing/src/main/java/io/druid/guice/ExtensionsConfig.java

```diff
@@ -33,10 +34,6 @@
   @NotNull
   private boolean searchCurrentClassloader = true;
 
-  @JsonProperty
-  @NotNull
-  private List<String> coordinates = ImmutableList.of();
```

Comment:
Not sure if I fully understand this. Are you worrying about people not being aware of one of the nodes hadn't been updated to the latest version? If that node fails on bootup then people will realize it's not updated properly. If that node just runs the old version silently, then failing on "specifying ```druid.extensions.coordinates```" will not help either since that node didn't pick up this code. Or did I miss something?